README.md

```diff
@@ -1,4 +1,54 @@
-rednoi
-======
+# RedNoi
 
-A Symfony project created on February 22, 2017, 7:06 pm.
+Overwhelmed? Let RedNoi helps you to overcome your information overload. 
+
+This tool is for advanced Twitter/TweetDeck users only, beginners might need to understand what Twitter/TweetDeck parameters are and how they work before using it.
+
+Rednoi uses possibilities of [The Search API][] where you can use the filters on public twitter lists. The results are then combined and serialized by time into one stream. So the final result provides the one data stream same like the Twitter and with filtered data of your specific lists.
```

Comment:
provides the one data stream same like... ➡️ provides one data stream, similar to standard Twitter, but with data filtered according to your specific needs.